chatterbox/.gitignore

```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
+
```

Comment:
Why 2 .gitignores?

Reply:
It's the project structure. we created chatterbox inside root folder but the one outside doesn't work and will be deleted.

Reply:
ok